code/datums/components/thermite.dm

```diff
@@ -3,8 +3,17 @@
 	var/amount
 	var/overlay
 
-	var/static/list/blacklist = typecacheof(/turf/closed/wall/mineral/diamond)
-	var/static/list/resistlist = typecacheof(/turf/closed/wall/r_wall)
+	var/static/list/blacklist = typecacheof(
+		/turf/closed/wall/mineral/diamond,
+		/turf/closed/indestructible,
+		/turf/open/lava,
```

Comment:
![image](https://user-images.githubusercontent.com/6209658/33110299-fcb1a3e0-cf14-11e7-8f88-52ca0359585a.png)

/turf/open/indestructible
/turf/open/space

Reply:
space just becomes space again, it still burns even with blacklist. It just doesn't get change turf'd

Reply:
I'll make an actual blocked list